packages/dao/contracts/test/Imports.sol

```diff
@@ -0,0 +1,8 @@
+pragma solidity 0.4.24;
+
+
+//TODO: This is added to use this contract in tests, I think this is stupid, need to think ho to change it
```

Comment:
delete this comment pls :)

Reply:
Deleted, I still think that's not an ideal solution though 😂